src/bun.zig

```diff
@@ -3461,6 +3467,9 @@ pub fn SliceIterator(comptime T: type) type {
 
 pub const Futex = @import("./futex.zig");
 
+// TODO: migrate
+pub const ArenaAllocator = std.heap.ArenaAllocator;
```

Comment:
what happened to `ArenaAllocator`?

Reply:
it was a 1:1 copy of the current standard library

Reply:
lol